### PR TITLE
Lidar tutorial

### PR DESCRIPTION
**Pull request by SmithB**
Here's my second try at an initial commit for the lidar tutorial.  I've saved a clean version of the notebook, and removed the login code that (I think) was causing an error.

### CODE REVIEW

File: book/_toc.yml [hunk 1]
```diff
@@ -47,6 +47,9 @@
             - file: tutorials/lis/1_exploring_lis_output
             - file: tutorials/lis/2_interactive_widgets
         - file: tutorials/gpr/gpr
+        - file: tutorials/lidar/index
+          #sections:
+            #- file: tutorials/lidar/ICESat2_tutorial
```
Comment (scottyhq) [inline book/_toc.yml:52]: @SmithB go ahead and uncomment this so that the notebook is executed

**Comment by scottyhq** [comment]
@SmithB icepyx and sliderule-python are now in the default image if you want to uncomment those cells and move forward with this. up to you how much content goes in, let me know when you'd like to merge it

**Comment by scottyhq** [comment]
looking good @SmithB , can you reduce the log output of sliderule (i see lots of `unable to process resource ATL03_20181110092841_06530106_003_01.h5: no elements`). also you'll have to use `%matplotib inline` for getting figure output on the website it seems

**Comment by SmithB** [comment]
I've made some more commits to the tutorial.  It's saved in clean form.  Would it be possible to merge this so that Clare has a directory to work in?

**Comment by SmithB** [comment]
I've made a few more updates.  Can this be merged?  Thanks!

**Review by scottyhq — APPROVED** [review]

**Comment by jomey** [comment]
@SmithB - Can you go over the typos highlighted under the 'files' type and address those?
https://github.com/snowex-hackweek/website/pull/81/files

Then we are good to merge

**Comment by SmithB** [comment]
In response to @joachim, I've updated the spelling on one of my variable names.  Let me know if there's anything else to be done.

**Comment by SmithB** [comment]
Should be done!  Thanks for calling that to my attention.

**Pull request merged**